ecosystem/sep-0001.md

```diff
@@ -6,8 +6,8 @@ Title: stellar.toml
 Author: stellar.org
 Status: Active
 Created: 2017-10-30
-Updated: 2020-12-11
-Version: 2.1.1
+Updated: 2021-01-27
+Version: 2.1.2
```

Comment:
This adds new functionality so we should increment the minor version instead of the patch version. The path version is only used when something needs fixing.
```suggestion
Version: 2.2.0
```